cli/flox/Cargo.toml

```diff
@@ -56,6 +56,7 @@ url.workspace = true
 uuid.workspace = true
 xdg.workspace = true
 flox-core.workspace = true
+sapling-renderdag = "0.1.0"
```

Comment:
nonblocking question: how did you choose this crate? Looks like it hasn't been touched in about a year, and it has no documentation. Probably good enough for now, but doesn't seem the greatest

Reply:
Makes sense 👍